marimbabot_description/urdf/marimbabot_ur5.urdf.xacro

```diff
@@ -130,7 +130,7 @@
   <joint name="world_marimba_joint" type="fixed">
     <parent link="world" />
     <child link = "marimba" />
-    <origin xyz="1.25 0.8 0.80" rpy="0.0 0.0 ${pi}" />
```

Comment:
What was the problem here? :)

Reply:
The marimba was too far away, so we where not able to reach certain black keys. This was solved in a hacky way by using approximate ik solutions, but it doesn't hurt to pull it a little closer and use the exact solutions.